HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -272,6 +272,16 @@ def replaceItemsInSequence(process, seqNames, itemsToReplace, replacingSequence)
 
     return process
 
+def customizeHLTforAlpakaPFSoA(process):
+
+    # avoid the conversion from SoA to legacy to SoA for the HCAL rechits
+    process.hltParticleFlowRecHitHBHESoA.producers[0].src = 'hltHbheRecoSoA'
```

Comment:
@kakwok @fwyzard seems we missed to protect here the cases where these modules do not exist (I.e. the other tables that are not `GRun`). This leads to crashes in the integration tests when trying to integrate this in the combined table. Can you provide a fix for this real quick?

Reply:
@mmusich does https://github.com/cms-sw/cmssw/pull/45484 work for you ?

Reply:
@fwyzard thank you. 

> does https://github.com/cms-sw/cmssw/pull/45484 work for you ?

 I think so. Martin was also reporting problems related to the Hcal local reco part of the customization concerning `hltTowerMakerForAllSerialSync`. Apparently the check at the top https://github.com/cms-sw/cmssw/blob/b08e6a8d58828b9f9000288e75e2f64794efd261/HLTrigger/Configuration/python/customizeHLTforAlpaka.py#L1106

does not cover all the use cases.